packages/manager/apps/container/src/payment-modal/PaymentModal.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
+import { Modal } from 'react-bootstrap';
```

Comment:
Finally use the modal of react-bootstrap ?

Reply:
Yes because it's already use for other modal.